src/applications/personalization/dashboard/tests/containers/DashboardApp.unit.spec.jsx

```diff
@@ -22,8 +25,34 @@ describe('<DashboardApp>', () => {
   });
 
   it('should render', () => {
-    const tree = SkinDeep.shallowRender(<DashboardApp {...props}/>);
+    const tree = SkinDeep.shallowRender(<DashboardApp {...defaultProps}/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.be.ok;
   });
+
+  it('should render verification state if LOA != 3', () => {
+    const tree = SkinDeep.shallowRender(<DashboardApp profile={{ loa: { current: 1 } }}/>);
+    expect(tree.toString()).to.contain('Verify your identity to access more Vets.gov tools and features');
+  });
+
+  it('should render MVI warning state if status not OK', () => {
+    const tree = SkinDeep.shallowRender(<DashboardApp profile={{ loa: { current: 3 }, status: 'ERROR' }}/>);
+    expect(tree.toString()).to.contain('We’re having trouble matching your information to our Veteran records');
+  });
```

Comment:
We could possibly move the MVI Warning state to its own component and check for its existence here, instead of a certain string value that could trip up tests during a quick content change someday.